util.py

```diff
@@ -0,0 +1,9 @@
+def strtobool(value):
+    value = value.lower()
+    if value in ["y", "yes", "t", "true", "on", "1"]:
```

Comment:
These comparisons are O(N) and always recreate the list. Make the list definition global and use a set instead of a list.